recipes/evalhyd-cpp/meta.yaml

```diff
@@ -0,0 +1,56 @@
+# name for conda package (i.e. `conda install -c conda-forge <project_name>`)
+{% set project_name = "evalhyd-cpp" %}
+# name for C++ library
+{% set library_name = "evalhyd" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "eacfa5ad3bad30cb3af7b5584a0c1183a89de109fd252d0d4cfca47db891fdae" %}
+
+package:
+  name: {{ project_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.irstea.fr/HYCAR-Hydro/evalhyd/{{ project_name }}/-/archive/v{{ version }}/{{ project_name }}-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true                  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.15
+    - make                    # [unix]
+    - ninja                   # [win]
+  host:
+    - xtl >=0.7.5
+    - xtensor >=0.24.6
+
+test:
+  commands:
```

Comment:
Please add tests for the presence of shared libs, absence of static ones.

Reply:
Could you help me here please? I am not sure how to do this.

Reply:
The link I gave above for the arrow-cpp feedstock has a nicely worked-out example of this (for several libraries; if there's only one you can drop the for loop)

Reply:
Thank you. I have added similar tests, but they seem to fail. So I must not be building the shared library properly?

Reply:
But that's because I create an interface library, isn't it?

Reply:
Sorry, I haven't looked at the library specifically, I was talking in general principles. If it's header only and there are no library artefacts, then that's fine and no further tests are needed